index.html

```diff
@@ -358,11 +352,11 @@
 <li>Let <var>deadline</var> be a time in the future until which the browser expects to remain idle.</li>
 <li>If <var>deadline</var> - <var>now</var> is greater than 50ms, then cap <var>deadline</var> by setting it to
 be <var>now</var> + 50ms.</li>
-<li>Let <var>docs</var> be the list of <a href="http://www.w3.org/TR/html5/browsers.html#fully-active">fully active</a> <code>Document</code> objects associated with the event loop in question, sorted arbitrarily.</li>
-<li>For every <var>document</var> in <var>docs</var>, in any order, perform the following steps:
+<li>Let <var>windows</var> be the <code>Window</code> object's list of <a href="https://www.w3.org/TR/html5/browsers.html#browsing-context">browsing context</a>'s <code>WindowProxy</code> objects associated with the event loop in question.</li>
```

Comment:
Stick with _docs_, and for each _doc_, look at "_doc_'s `Window` object's list of..."